arch/loongarch/slide_hash_lsx.c

```
@@ -47,8 +47,15 @@ Z_INTERNAL void slide_hash_lsx(deflate_state *s) {
     assert(((uintptr_t)s->head & 15) == 0);
     assert(((uintptr_t)s->prev & 15) == 0);
 
-    slide_hash_chain(s->head, HASH_SIZE, xmm_wsize);
-    slide_hash_chain(s->prev, wsize, xmm_wsize);
+    slide_hash_chain_static(s->head, HASH_SIZE, xmm_wsize);
+    slide_hash_chain_static(s->prev, wsize, xmm_wsize);
+}
+
+Z_INTERNAL void slide_hash_head_lsx(deflate_state *s) {
```

Cppcheck (2.21.0)

[style] The function 'slide_hash_head_lsx' is never used.

(unusedFunction)